src/App.js

```diff
@@ -2,17 +2,16 @@
 
 import Alert from "@material-ui/lab/Alert";
 import AlertTitle from "@material-ui/lab/AlertTitle";
-import ArrowBack from "@material-ui/icons/ArrowBack";
 import { Button, makeStyles, MuiThemeProvider } from "@material-ui/core";
 import CovidAppointmentTable from "./CovidAppointmentTable";
 import Drawer from "@material-ui/core/Drawer";
+import FilterListIcon from "@material-ui/icons/FilterList";
 import FilterPanel, { getZipCodeCookie } from "./components/FilterPanel";
 import {
     filterData,
     getAppointmentData,
 } from "./services/appointmentData.service";
 import Grid from "@material-ui/core/Grid";
-//import { Alert, AlertTitle } from "@material-ui/lab";
```

Comment:
Could you put this back in, please? It's so it's easy to uncomment when we want to put an alert back in.